lib/mojito/headers.ex

```diff
@@ -199,4 +199,29 @@ defmodule Mojito.Headers do
     auth64 = "#{username}:#{password}" |> Base.encode64()
     {"authorization", "Basic #{auth64}"}
   end
+
+  @doc ~S"""
+  Convert non string values to string where is possible.
+
+  Example:
+
+      iex> Mojito.Headers.convert_values_to_string([{"content-length", 0}])
+      [{"content-length", "0"}]
+  """
+  @spec convert_values_to_string(headers) :: headers
+  def convert_values_to_string(headers) do
+    convert_values_to_string(headers, [])
+  end
+
+  defp convert_values_to_string([], converted_headers),
+    do: Enum.reverse(converted_headers)
+
+  defp convert_values_to_string([{name, value} | rest], converted_headers)
+       when is_number(value) do
```

Comment:
I think it makes sense to do the `to_string` conversion all the time, that way we will stringify atoms as well.

Reply:
I think we need to pick datatypes that we want convert. What if it a list `[1,2,3]`? Then better to pass it as is mo Mint and let it fail.

What do you think?

Reply:
It's not meaningful to pass a list of anything as a header value. If someone does that, I am OK with things blowing up here instead of inside Mint.

Reply:
But things won't blow up if we convert the list to string. The mistake will go unnoticed and will more time to debug what's wrong.

Are you really sure?

Reply:
Ah true. Let's keep this as-is then.

Reply:
Left as is, but added conversions for atoms. Atoms seem like a reasonable datatype for it.